bin/reth/src/node/mod.rs

```diff
@@ -263,19 +269,32 @@ impl<Ext: RethCliExt> NodeCommand<Ext> {
         // setup the blockchain provider
         let factory = ProviderFactory::new(Arc::clone(&db), Arc::clone(&self.chain));
         let blockchain_db = BlockchainProvider::new(factory, blockchain_tree.clone())?;
-
         let blob_store = InMemoryBlobStore::default();
-        let transaction_pool = reth_transaction_pool::Pool::eth_pool(
+
+        let validator = if let Some(ref trusted_setup_file) = self.trusted_setup_file {
+            let kzg_settings = self.load_kzg_settings(trusted_setup_file)?;
+            info!(target: "reth::cli", "KZG trusted setup file loaded");
+
+            TransactionValidationTaskExecutor::eth_with_kzg_settings_and_tasks(
```

Comment:
I don't like this if let here because this makes it too verbose for my taste,

the better solution here would be adding a new function that returns `Arc<KzgSettings>`

so we only need to call `TransactionValidationTaskExecutor::eth_with_` once

btw @Rjected if the fallback is MAINNET_KZG then this is problematic for other preconfigured chains like sepolia?
or do they use the same trusted setup?

Reply:
>  or do they use the same trusted setup?

This is not defined yet, it's unlikely they would be different though